### PR TITLE
Improves template placeholders

### PR DESCRIPTION
**Pull request by JosephSalisbury**
Makes the template placeholders more explicit

### CODE REVIEW

File: .github/rfc-format/check-rfc-format.py
Comment (AndiDog) [inline .github/rfc-format/check-rfc-format.py:245]: Fine, but let's do this instead at the beginning of the loop so we bail out early

```
if entry_name == 'template':
    continue
```

**Review by snizhana-dynnyk — APPROVED** [review]

**Comment by JosephSalisbury** [comment]
@snizhana-dynnyk we're good to merge here once @AndiDog approves 🚀

**Review by AndiDog — APPROVED** [review]

**Pull request merged**
